scarb/src/ops/lint.rs

```diff
@@ -154,16 +155,17 @@ pub fn lint(opts: LintOptions, ws: &Workspace<'_>) -> Result<()> {
                         .flat_map(|module_id| db.module_semantic_diagnostics(*module_id).ok())
                         .collect();
 
-                    let renderer = Renderer::styled();
-
                     let diagnostics = diags
                         .iter()
                         .flat_map(|diags| {
                             let all_diags = diags.get_all();
-                            all_diags.iter().for_each(|diag| {
-                                ws.config()
-                                    .ui()
-                                    .print(format_diagnostic(diag, &db, &renderer))
+                            all_diags.iter().for_each(|diag| match diag.severity() {
```

Comment:
Shouldn't we move https://github.com/software-mansion/scarb/blob/a309d7ca3108c44082a50d6ac48295783707eecf/scarb/src/compiler/helpers.rs#L43 to separate function and use it here?